internal/worker/worker.go

```diff
@@ -156,7 +156,7 @@ func (w *BaseWorker) Start(ctx context.Context) error {
 			}
 			// executeCommand executes the command and return the response back to the client
 			func(errChan chan error) {
-				execCtx, cancel := context.WithTimeout(ctx, 6*time.Second) // Timeout set to 6 seconds for integration tests
+				execCtx, cancel := context.WithTimeout(ctx, 1000*time.Second) // Timeout set to 6 seconds for integration tests
```

Comment:
nit: comment here mentions different timeout, could lead to confusion around the actual timeout needed

Reply:
Thanks for pointing that out, I was testing something so increased that up to 1000, will bring it back to 6.